api/src/pcapi/core/mails/backends/logger.py

```diff
@@ -15,24 +15,20 @@ class LoggerBackend(BaseBackend):
     """A backend that logs e-mail instead of sending it.
 
     It should be used for local development, and on testing/staging
-    when performing load tests when we don't want to overload Mailjet.
+    when performing load tests when we don't want to overload Sendinblue.
     """
 
     def _send(
         self,
         recipients: typing.Iterable[str],
-        data: typing.Union[dict, SendinblueTransactionalEmailData, SendinblueTransactionalWithoutTemplateEmailData],
+        data: typing.Union[SendinblueTransactionalEmailData, SendinblueTransactionalWithoutTemplateEmailData],
     ) -> MailResult:
-        if isinstance(data, SendinblueTransactionalEmailData):
-            data = asdict(data)
-            mail_server = "Sendinblue"
-        elif isinstance(data, SendinblueTransactionalWithoutTemplateEmailData):
-            data = asdict(data)
+        if not isinstance(data, SendinblueTransactionalEmailData):
             mail_server = "Sendinblue with no template"
-        else:
-            mail_server = "Mailjet"
+        mail_server = "Sendinblue"
         recipients = ", ".join(recipients)
-        logger.info("An e-mail would be sent via %s to=%s: %s", mail_server, recipients, data)
-        result = MailResult(sent_data=data, successful=True)
+        sent_data = asdict(data)
+        logger.info("An e-mail would be sent via %s to=%s: %s", mail_server, recipients, sent_data)
```

Comment:
(détail) plus besoin de la variable `mail_server` :

```suggestion
        logger.info("An e-mail would be sent via SendinBlue to=%s: %s", recipients, sent_data)
```